docs/user-guide/cli-configure-scs-on-headless-linux-os.md

```diff
@@ -58,42 +72,85 @@ When you are using SSH or TTY to log in to Linux, you can configure the Gnome ke
+For instructions to set up the credential manager binaries for Red Hat Enterprise Linux (RHEL) V8.X and Ubuntu z/Linux systems, refer to this section. See [Configuring RHEL V7.X](#configuring-rhel-v7x) for instructions specific to that distribution.
 
-1. Install the following Linux packages on the z/Linux system:
-    - make
-    - gcc-c++ (sometimes available as g++)
-    - gnome-keyring
-    - libsecret (sometimes available as libsecret-1-0)
-    - libsecret-devel (sometimes available as libsecret-1-dev)
+To install and build the credential storage binaries on z/Linux RHEL V8.X and Ubuntu systems:
+
+1. Use the command prompt to install the following Linux packages on the z/Linux system:
+    - `make`
+    - `gcc-c++` (sometimes available as `g++`)
+    - `gnome-keyring`
+    - `libsecret` (sometimes available as `libsecret-1-0`)
+    - `libsecret-devel` (sometimes available as `libsecret-1-dev`)
     - Python 3.6 or later
 
-    **Note:** If you are installing the Linux packages on a z/Linux system, the system where you are configuring SCS might require Internet access. When a site hosts its own package repositories, the repositories might not contain all of the packages that are required to configure the SCS. In this scenario, the z/Linux system requires Internet access to install the required packages.
+    :::note
+    
+    If you are installing the Linux packages on a z/Linux system, the system where you are configuring secure credential storage might require Internet access. When a site hosts its own package repositories, the repositories might not contain all of the packages that are required to configure the secure credential storage. In this scenario, the z/Linux system requires Internet access to install the required packages.
+
+    :::
+
+2. If you are configuring secure credential storage on a Ubuntu z/Linux operating system, install Zowe CLI.
 
-2. If you are configuring SCS on a Ubuntu z/Linux operating system, no further action is required. You can now install Zowe CLI. For all other platforms (RHEL), continue to the next step.
+    For all other platforms (RHEL), continue to the next step.
 
 3. Enable the `rhel-#-for-system-z-optional-rpms` repository to download libsecret-devel.
 
-    Replace ***#*** with the major version of RHEL that is running on the z/Linux system.
+    If your license entitles you to this repository, open the command prompt and issue the following command to enable it:
+    
+    ```
+    subscription-manager repos —-enable rhel-#-for-system-z-optional-rpms
+    ```
 
-    If your license entitles you to this repository, issue the following command to enable it:
+    Replace `#` with the major version of RHEL that is running on the z/Linux system.
+
+4. [Unlock the keyring manually](#unlocking-the-keyring-manually) or [unlock the keyring automatically](#unlocking-the-keyring-automatically) to load and store credentials **[correct?]**.
```

Comment:
@adam-wolfe @zFernand0 is "to load and store creds" correct here?

Reply:
Yeah, I think it's ok to mention the reason why someone want to unlock the keyring, and those (load and secure) are the primary operations that Zowe CLI performs on the keyring 😋